devnet/devnet.go

```diff
@@ -85,6 +91,7 @@ func setDefaultWalletCmd(ctx context.Context) {
 		if err != nil {
 			continue
 		}
+		// TODO: stop once we've set the default wallet once.
```

Comment:
setting the default wallet could be faster and less hacky, but it doesn't feel urgent right now. I left TODOs.